### ISSUE TITLE
Dropbox plugin: hundreds/thousands of action-down-* and temp-action-* files

### ISSUE THREAD

**Issue by sanjay** [syncany/syncany#500]
Syncing a folder to a Dropbox folder (and no other machine syncing to the same folder), Syncany creates a handful of zero-byte action-down-* files in action/ and temp-action-* in the main directory every few seconds.  I discovered this issue when I was no longer syncing files because there were hundreds of thousands of files in the directory.  This apparently made Dropbox choke.

**Comment by sanjay** [comment]
Seems that this issue is being exacerbated by how Dropbox works.  I deleted the parent folder and then tried again and it seems okay for now.  Perhaps some robustness needs to be added into the Dropbox plugin to deal with the timing delays that sometimes happens with the Dropbox API in propagating changes/updates.

**Comment by pimotte** [comment]
I suspect the actual issue here is the fact that we do an action every few seconds, and apperent do not delete the action files. Do you have any syncany logs that might help us debug this?

**Comment by sanjay** [comment]
Where are the log files?  Not sure I have any at this point.  I killed off the directory in Dropbox and then re-setup everything.  Now I don't have any issues with hundreds/thousands of zero-byte files.  So not sure where the problem lies - Dropbox or Syncany.

**Comment by pimotte** [comment]
There are logs in the .syncany/logs folder in your client, but I suspect you have deleted those. If you used the daemon (and/or GUI) there will be logs in %AppData%\Syncany\ or ~/.config/syncany/.

Also, was your syncany client in your Dropbox folder, or was it also being synced by another piece of syncing software?

**Comment by sanjay** [comment]
yeah, I've wiped that folder - oops.  That said, the logs in .config are still there but the main log is 13 megs.  Want me to send that somehow (I'll peel off some of the folders that aren't related to this issue).  This was through the Dropbox plugin with Syncany so Syncany should be managing all of the interactions.  It isn't that the Syncany client is in Dropbox either - all through the Dropbox API (presumably).

**Comment by pimotte** [comment]
If you could mail us that log (team@syncany.org) with any info you don't want to send/think is irrelevant redacted, that would be great:)

**Comment by sanjay** [comment]
Sent the email an hour or so ago.  Good luck debugging.
